### DISCUSSION TITLE
posts/01-frontend/WebRTC/02-p2p

### DISCUSSION THREAD

**Discussion by giscus** [wangrongding/wangrongding.github.io#10] [Announcements]
# posts/01-frontend/WebRTC/02-p2p

荣顶 - 个人博客

https://www.fedtop.com/posts/01-frontend/WebRTC/02-p2p.html

**Comment by zuoyifeng** [comment]
大佬你好，下一篇文章示例好像无法实现加入提示房间已满，是不是应该加上用户名id问题

**Reply by wangrongding** [reply]
任意房间号都无法加入吗？我限制了每个房间两人，可能里面有人了
